### PR TITLE
Moved Twitter feed to updates section

### LINKED ISSUE #10: Layout of "home" page is lopsided
This depends on #8 getting merged in.

The left side of the page is very long, while the right side consists of two headers and a small paragraph. Putting the twitter widget near the "Current News" section could fix the problem.

### PR DESCRIPTION
**Pull request by stevelyall**
Should resolve #10 ?
Also added script to keep the copyright year up to date

### CODE REVIEW

**Comment by mitchhentges** [comment]
Hmm, I'd like to have the Facebook link back in, personally. There's still some work that needs to be done to make it fit in better (it's taking up the whole right side of the page, yet it's styled like a widget) and there is a space at the bottom of the page where the footer should be.
spaces

**Comment by mitchhentges** [comment]
@stevelyall asked IRL: how do wat yes hold on [site](https://rawgit.com/stevelyall/trucsclub.github.io/master/index.html)

**Comment by stevelyall** [comment]
Better?

screenshot 2015-01-12 16 17 41

File: index.html [hunk 1]
```diff
@@ -118,18 +124,20 @@ <h3 class="widget-title">Become a member!</h3>
...
-          </script>
+        
         </aside>
 			</div><!-- /Sidebar -->
 		</div>
     </div>
     
-    <footer class="footer"><p>Copyright 2014 Computer Science Club</p></footer>
+    <footer class="footer"><p>Copyright &copy;
+        <script language="javascript" type="text/javascript">
```
Comment (chrisfosterelli) [inline index.html]: Just FYI, in HTML5 if you specify `<script>` then `language="javascript"` and `type="text/javascript"` are automatically assumed, so it's still valid to leave them out. What you have works as well though of course :)

**Comment by chrisfosterelli** [comment]
I think there is lots of value in keeping the facebook link in as well.

**Comment by chrisfosterelli** [comment]
But yeah, LGTM :+1:

**Comment by stevelyall** [comment]
How do we feel about this change? I know we're all working on other stuff but it would be cool to get the main page looking a little more finished :)

**Pull request merged**